<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditServeurMonoWidget</class>
 <widget class="QWidget" name="AuditServeurMonoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditServeurMonoWidget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Num√©ro de port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_Ports">
            <property name="minimum">
             <number>1025</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Lancement">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Lancement Serveur</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Informations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_Messages"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Quitter">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditServeurMonoWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
